js_modules/dagster-ui/packages/ui-core/src/jobs/JobsPageContent.tsx

```diff
@@ -157,7 +157,7 @@ export const JobsPageContent = () => {
             icon="search"
             value={searchValue}
             rightElement={
-              showSearchSpinner ? <SearchInputSpinner tooltipContent="Loading jobs…" /> : undefined
+              loading ? <SearchInputSpinner tooltipContent="Loading jobs…" /> : undefined
```

Comment:
This one for example seems like it should remain until the workspace data is up to date